src/guacd-docker/sbin/start.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+##
+## @fn start.sh
+##
+## Automatically configures and starts GUACD.
+##
+
+
+# guacd default value if not overridden
+
+if [ -z "$GUACD_LISTEN_ADDRESS" ]; then
+    export GUACD_LISTEN_ADDRESS=0.0.0.0
+fi
+if [ -z "$GUACD_PORT" ]; then
+    export GUACD_PORT=4822
+fi
+if [ -z "$GUACD_LOG_LEVEL" ]; then
+    export GUACD_LOG_LEVEL=info
+fi
+
+args=( -b "${GUACD_LISTEN_ADDRESS}" -L "${GUACD_LOG_LEVEL}" -l "${GUACD_PORT}" -f )
+
+# guacd certificate files
+if [ -n "$GUACD_CERTIFICATE_CRT" ]; then
+    args+=( -C "$GUACD_CERTIFICATE_CRT" )
+fi
+if [ -n "$GUACD_CERTIFICATE_KEY" ]; then
+    args+=( -K "$GUACD_CERTIFICATE_KEY" )
+fi
```

Comment:
Does it make sense to process these individually? I believe they are both required for SSL to function, so seems like this check should be for both of them, and fail if one is defined without the other.

Reply:
makes sense that they are both required for SSL to work.